packages/@uppy/drag-drop/src/index.js

```diff
@@ -142,6 +142,7 @@ module.exports = class DragDrop extends Plugin {
     const restrictions = this.uppy.opts.restrictions
     return (
       <input
+        id={'uppy-input-' + this.id}
```

Comment:
we might want to do `this.uppy.id + this.id` in case someone has two instances of uppy with drag-drop

Reply:
Totally, thanks!